code/modules/spacepods/spacepod.dm

```diff
@@ -54,12 +68,39 @@
 	var/move_delay = 2
 	var/next_move = 0
 
+/obj/spacepod/proc/apply_paint(mob/user as mob)
+	var/part_type
+	var/part = input(user, "Choose part", null) as null|anything in list("Lights","Rim","Paint","Windows","Cancel")
+	if(part == "Lights")
+		part_type = LIGHT
+	if(part == "Rim")
+		part_type = RIM
+	if(part == "Paint")
+		part_type = PAINT
+	if(part == "Windows")
+		part_type = WINDOW
+	if(part == "Cancel")
```

Comment:
Do a `!part` check here and remove the `cancel` option from the list, BYOND already gives you one.